docs/installation-and-operations/installation/docker/README.md

```diff
@@ -266,7 +266,34 @@ First, add a `.env` file with some variable:
 OPENPROJECT_HTTPS=true
 ```
 
-And the  `docker-compose` command will automatically pick it up. You can also specify multiple files if you have different configurations you want to test.
+And then you'll need to pass the environment variable to the respective containers you want to set it one. For most OpenProject environment variables, this will be for `x-op-app`:
```

Comment:
minor typo

```suggestion
And then you'll need to pass the environment variable to the respective containers you want to set it on. For most OpenProject environment variables, this will be for `x-op-app`:
```